src/scss/utilities/_positioning.scss

```diff
@@ -0,0 +1,26 @@
+
+// Will push a flex item to the top
+.ec-push-t {
+  margin-bottom: auto;
+}
+
+// Will push a flex item to the bottom
+.ec-push-b {
+  margin-top: auto;
+}
+
+// Will push a flex item to the left
+.ec-push-l {
+  margin-right: auto;
+}
+
+// Will push a flex item to the right
+.ec-push-r {
+  margin-left: auto;
+
+  &-from-1024 {
```

Comment:
I would use `@` in the name for responsive selectors, e.g. `.ec-push-r@xl`

Reply:
I've usually approached this differently, and maybe we can move this discussion offline as it can turn philosophical even,

I namespace with the responsive utility prefix, for use with utility classes. At the end it's just adding the modifier in the prefix rather than as a suffix, but it helps if you want to loop through the viewports, in sass pseudocode:

```
@each $viewport, $breakpoint in $viewports {
  @media (--#{$viewport}) {
    // component modifiers loop
      .r-#{$viewport}-{{component}}--{{modifier}} {
        // component mixin
      }
    }
  }
}
```

in any case first is the discussion if we want viewport modifiers for components

Reply:
Should be fine for modal as it always has the viewport as a parent so the behavior is always be the same.